runtime/src/main/java/org/corfudb/runtime/view/StreamsView.java

```diff
@@ -93,38 +93,10 @@ public IStreamView copy(UUID source, UUID destination, long timestamp) {
      * @return The address this
      */
     public long write(Set<UUID> streamIDs, Object object) {
-        return acquireAndWrite(streamIDs, object, t -> true, t -> true);
-    }
-
-    public void writeAt(TokenResponse address, Set<UUID> streamIDs, Object object) throws OverwriteException {
-        Function<UUID, Object> partialEntryFunction =
-                object instanceof IDivisibleEntry ? ((IDivisibleEntry)object)::divideEntry : null;
-        runtime.getAddressSpaceView().write(address.getToken(), streamIDs,
-                object, address.getBackpointerMap(), address.getStreamAddresses(), partialEntryFunction);
-    }
-
-    /**
-     * Write an object to multiple streams, retuning the physical address it
-     * was written at.
-     * <p>
-     * Note: While the completion of this operation guarantees that the append
-     * has been persisted, it DOES NOT guarantee that the object has been
-     * written to the stream. For example, another client may have deleted
-     * the stream.
-     *
-     * @param object The object to append to the stream.
-     * @return The address this
-     */
-    public long acquireAndWrite(Set<UUID> streamIDs, Object object,
```

Comment:
Is there a particular reason you removed the callbacks? It still seems to be potentially useful to know when a token was acquired and when you were overwritten?